clappr/clappr-ios
# Swift 4 Update

### Goal
Upgrade the code to Swift 4 with its dependencies.

### How to test

- Check the changes
- Check the build for `iOS` and `tvOS`
- Run all our tests
- Go to `steve-tvOS` and `steve` projects and check the Swift 4 Update PR.
